protos/camera/camera.proto

```diff
@@ -296,6 +296,7 @@ message VideoStreamSettings {
     uint32 vertical_resolution_pix = 3; // Vertical resolution (in pixels)
     uint32 bit_rate_b_s = 4; // Bit rate (in bits per second)
     uint32 rotation_deg = 5; // Video image rotation (clockwise, 0-359 degrees)
+    float horizontal_fov_deg = 7; // Horizontal fov in degrees
```

Comment:
@JonasVautherin @markusachtelik is 7 above 6 on purpose here? Something to do with trimming?

Reply:
leftover from some experimenting, will change